avocado/core/safeloader.py

```diff
@@ -101,6 +101,33 @@ def is_docstring_tag_disable(docstring):
     return result == 'disable'
 
 
+def is_docstring_test_tags(docstring):
+    """
+    Checks if there's an avocado tag that tags a test in a certain categories
+
+    :rtype: bool
+    """
+    result = get_docstring_tag(docstring)
+    if result is not None:
+        return result.startswith('tags=')
+    return False
+
+
+def get_docstring_test_tags(docstring):
+    """
+    Returns the test categories based on a `:avocado: tags=category` docstring
+
+    :rtype: set
+    """
+    if not is_docstring_test_tags(docstring):
+        return []
```

Comment:
If we return `None` here, we can distinguish between not-tagged and empty set, right?